scripts/test/large.py

```diff
@@ -0,0 +1,83 @@
+from modules import utils
+from modules.logger import log
+from unittest import TextTestRunner
+
+import sys
+import unittest
+
+
+class PsutilCollectorLargeTest(unittest.TestCase):
+
+    def setUp(self):
+        # set and download required binaries (snapd, snapctl, plugins)
+        self.binaries = utils.set_binaries()
+        utils.download_binaries(self.binaries)
+
+        log.debug("Starting snapd")
+        self.binaries.snapd.start()
+        if not self.binaries.snapd.isAlive():
+            self.fail("snapd thread died")
+
+        log.debug("Waiting for snapd to finish starting")
+        if not self.binaries.snapd.wait():
+            log.error("snapd errors: {}".format(self.binaries.snapd.errors))
+            self.binaries.snapd.kill()
+            self.fail("snapd not ready, timeout!")
+
+    def test_psutil_collector_plugin(self):
+        # load psutil collector
+        loaded = self.binaries.snapctl.load_plugin("snap-plugin-collector-psutil")
+        self.assertTrue(loaded, "psutil collector loaded")
+
+        # check available metrics, plugins and tasks
+        metrics = self.binaries.snapctl.list_metrics()
+        plugins = self.binaries.snapctl.list_plugins()
+        tasks = self.binaries.snapctl.list_tasks()
+        self.assertGreater(len(metrics), 0, "Metrics available {} expected {}".format(len(metrics), 0))
+        self.assertEqual(len(plugins), 1, "Plugins available {} expected {}".format(len(plugins), 1))
+        self.assertEqual(len(tasks), 0, "Tasks available {} expected {}".format(len(tasks), 0))
+
+        # check config policy for metric
+        rules = self.binaries.snapctl.metric_get("/intel/psutil/vm/free")
+        self.assertEqual(len(rules), 0, "Rules available {} expected {}".format(len(rules), 0))
+
+        # create and list available task
+        task_id = self.binaries.snapctl.create_task("/snap-plugin-collector-psutil/scripts/docker/large/psutil-task.yml")
+        tasks = self.binaries.snapctl.list_tasks()
```

Comment:
`self.binaries.snapctl.list_tasks()` called twice , first in line 35. any reason ?